src/internal/transport/AbstractRequestProcessor.ts

```diff
@@ -54,14 +59,18 @@ export abstract class AbstractRequestProcessor implements RequestProcessor {
                 mappedParameters: command.mapped_parameters,
                 secrets: command.secrets,
             };
-            const ctx: HandlerContext & AutomationContextAware = {
+            const ctx: any = {
```

Comment:
Why detype here?

Reply:
Actually you're right, I don't think that is need any longer. I can change that back.